extensions/vertx/runtime/src/main/java/io/quarkus/vertx/runtime/GenericMessageCodec.java

```diff
@@ -0,0 +1,67 @@
+package io.quarkus.vertx.runtime;
+
+import java.lang.reflect.Method;
+
+import org.jboss.logging.Logger;
+
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.eventbus.MessageCodec;
+import io.vertx.core.json.JsonObject;
+
+public class GenericMessageCodec implements MessageCodec<JsonObject, JsonObject> {
```

Comment:
This would be pretty expensive. In the case of local delivery, don't serialize to Json.

The codec should be `Object,Object`.

Reply:
Ok but I have a buffer, how do you want to serialize them on it ? 
If I don't use Json, I can serializable (ObjectOutputStream), or any other kind of conversion to byte array. because the signature use a buffer.  no ?

Reply:
i push a version based on serialization. If you are ok, i will add unit test after the implementation is ok for you

Reply:
Is it ok for you @cescoffier ?